### ISSUE TITLE
Curl error

### ISSUE THREAD

**Issue by jericopulvera** [hakanersu/codecourse-downloader#11]
I'm getting this error but after 2nd try download has started.

What causes this?

  [GuzzleHttp\Exception\RequestException]                                      
  cURL error 18: transfer closed with outstanding read data remaining (see ht  
  tp://curl.haxx.se/libcurl/c/libcurl-errors.html)

**Comment by DONSA** [comment]
Have no way to test it.
Try to replace Guzzle client instantiation at src/Remote.php to this:
```
$this->guzzle = new Client([
    'base_uri' => getenv('BASE_URL'),
    'curl' => [
        CURLOPT_TCP_KEEPALIVE => 900,
        CURLOPT_TCP_KEEPIDLE => 900
    ]
]);
```

**Comment by robfrancken** [comment]
I had this issue too on my Windows machine. Updating the guzzle client didn't work for me. It's not a solution, but to get around the issue I installed the downloader on an Ubuntu VM. Seems to be working fine there. I did get the following error after trying to run it for the first time though:

`[Symfony\Component\Debug\Exception\FatalThrowableError]  Class 'DOMDocument' not found `

Running `sudo apt-get install php-xml` should fix it.
